spec/helpers/logging_level_helper.rb

```diff
@@ -0,0 +1,8 @@
+# encoding: utf-8
+
+ENV["LOG_AT"].tap do |level|
```

Comment:
is this documented somewhere so that future devs know about this if useful?

Reply:
Not really. I suppose it should be in devutils but until now I have been reluctant to make changes in that direction.
It is useful in this plugin with --fail-fast and trace logging to see what is going on. Before, the tests might fail in the suite but not when run individually.
I am not sure how useful it is to other plugins that don't have extensive trace logging.

Reply:
sure - please don't touch devutils if only for getting rid of it. What I was saying is that since this could be useful to other developers that will work on this plugin and will need to run tests, it could be documented somewhere like a local CONTRIBUTING  file or unsure where else but the idea is that if it can be useful to other, they must know about it in some way?

Reply:
I understand now. I will add something when building the final changelog PR.